native/explorer/src/series.rs

```diff
@@ -1567,6 +1567,26 @@ pub fn s_split(s1: ExSeries, by: &str) -> Result<ExSeries, ExplorerError> {
     Ok(ExSeries::new(s2))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
```

Comment:
Series can be converted into lazy frame and the column/series can be manipulated using expression. That way we can avoid using ChunkedArrays.

```rust
let s2 = s1
        .clone_inner()
        .into_frame()
        .lazy()
        .select([col(s1.name())
            .str()
        // ops - mirroring expr_split_into
            .alias("res")])
        .collect()?
        .column("res")?
        .clone();
Ok(ExSeries::new(s2))
```